gammapy/scripts/catalog_query.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ..utils.scripts import get_parser
+
+__all__ = ['catalog_query']
+
+def main(args=None):
+    parser = get_parser(catalog_query)
+    parser.add_argument('--catalog',
```

Comment:
Use the argparse choices option here:
https://docs.python.org/3/library/argparse.html#choices